Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -138,29 +138,23 @@ public class CarPlayManager: NSObject {
         NotificationCenter.default.addObserver(self,
                                                selector: #selector(navigationCameraStateDidChange(_:)),
                                                name: .navigationCameraStateDidChange,
-                                               object: carPlayNavigationViewController?.navigationMapView?.navigationCamera)
+                                               object: carPlayMapViewController?.navigationMapView.navigationCamera)
```

Comment:
I think it was an oversight on my side. Looks like `CarPlayNavigationViewController` will never be available at the time of subscription, this means that subscription will not be working.